server/sonar-server/src/main/java/org/sonar/server/computation/task/step/ExecuteTaskInitExtensionsStep.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.computation.task.step;
+
+import org.sonar.api.ce.ComputeEngineSide;
+
+@ComputeEngineSide
+public class ExecuteTaskInitExtensionsStep implements ComputationStep {
+
+  private final TaskInitExtension[] extensions;
+
+  public ExecuteTaskInitExtensionsStep(TaskInitExtension[] extensions) {
+    this.extensions = extensions;
+  }
+
+  public ExecuteTaskInitExtensionsStep() {
```

Comment:
could be useful for new comers to add comment that this constructor is required by Pico when there is no TaskInitExtension